INTERPRETING.md

```diff
@@ -64,16 +64,15 @@ properties of the global scope prior to test execution.
         6. Return Completion(status).
 
   - **`global`** - a reference to the global object on which `$262` was initially defined
-  - **`uncallableAndIsHTMLDDA`** - a function that returns an object *`obj`* for
-    which [Call](https://tc39.github.io/ecma262/#sec-call)(*`obj`*, *any value*, «»)
-    throws a `TypeError`.  (The value of [IsCallable]()(*`obj`*) is unspecified:
-    a callable *`obj`* that throws a `TypeError` or an uncallable *`obj`* works
-    equally well.)  In hosts supporting the
-    [IsHTMLDDA](https://tc39.github.io/ecma262/#sec-IsHTMLDDA-internal-slot)
-    internal slot, *`obj`* must also have such a slot.  (These highly specific
-    behaviors are entirely motivated by the very few tests that use this.  Read
-    them for an explanation.)  Tests that use this function should be marked as
-    using the `uncallableAndIsHTMLDDA` feature.
+  - **`fakeDocumentAll`** - an object that 1) optionally has an [[IsHTMLDDA]]
```

Comment:
I would like to remove "optionally", personally. It should be on the hosts to decide whether or not they can provide this API - by analogy, not all hosts can provide "DetachArrayBuffer", but the hook still just says it is "a function which implements the DetachArrayBuffer abstract operation", no optionality about it.

And hosts which *can't* provide this API don't really need to run these tests anyway. From my perspective there's no point in allowing this API to be fulfilled by `{}`; it's just going to confuse test consumers about what is expected of them.

Reply:
I agree with this. @jswalden if you prefer I can fix/change this in a follow up.

Reply:
Yeah, that's fair.  Means SpiderMonkey's going to have to change before I can run these tests come next import, but that's easy enough to do.  I'm kinda more partial still to `fakeDocumentAll` for a name, but eh.  Changed the name as well in the latest update to the branch.